easybuild/easyconfigs/p/p4vasp/p4vasp-0.3.29-intel-2015b-Python-2.7.10.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'ConfigureMake'
+
+name = 'p4vasp'
+version = '0.3.29'
+versionsuffix = '-Python-2.7.10'
+
+homepage = 'http://www.p4vasp.at/'
+description = """Visualization suite for VASP"""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+
+# download via http://www.p4vasp.at/index.php/downloads
+sources = [SOURCE_TGZ]
+
+dependencies = [
+    ('PyGTK', '2.24.0', versionsuffix),
+]
+
+skipsteps = ['configure', 'build']
+
+# redefine $HOME to specify installation prefix O_o
+preinstallopts = "HOME=%(installdir)s make local &&"
```

Comment:
well, it is local, since the install command is run in a subshell anyway, and the change to `$HOME` is really only there for the `make local` command (no `export`, not a separate command to redefine `$HOME` but part of the `make local` command being run)